clloader/datasets/continuumDataset.py

```diff
@@ -0,0 +1,96 @@
+from typing import Callable, List, Tuple, Union
+
+import torch
+import numpy as np
+from clloader.viz import plot
+from PIL import Image
+from torchvision import transforms
+from torch.utils.data import Dataset as TorchDataset
+
+
+class ContinuumDataset(TorchDataset):
```

Comment:
It's not really a "continuum dataset" but more a "task dataset" here, isn't it?

Reply:
Oui j'ai compris ça après, je vais changer